ci/setter.sh

```diff
@@ -21,4 +21,4 @@ pushd python/ && python setup.py install && cd ..
 flake8 client/ --count --select=E9,F63,F7,F82,E302,W391,W292 --show-source --statistics
```

Comment:
Renamed this file to `ci/test.sh`.

Reply:
fixed in 366fff82a1e6a5d423e57a770b430cf2306c4240